src/cgame/cg_view.cpp

```diff
@@ -1414,10 +1414,27 @@ static void CG_CalcColorGradingForPoint( vec3_t loc )
 		}
 	}
 
-	cg.refdef.gradingWeights[0] = 0.0f;
-	cg.refdef.gradingWeights[1] = haveGlobal ? ( 1.0f - totalWeight ) : ( selectedWeight[0] / totalWeight );
-	cg.refdef.gradingWeights[2] = totalWeight == 0.0f ? 0.0f : selectedWeight[1] / totalWeight;
-	cg.refdef.gradingWeights[3] = totalWeight == 0.0f ? 0.0f : selectedWeight[2] / totalWeight;
+	if ( totalWeight )
+	{
+		cg.refdef.gradingWeights[0] = 0.0f;
+		cg.refdef.gradingWeights[1] = selectedWeight[0];
```

Comment:
Why not do the divisions here?

Reply:
This code will be obsoleted by: https://github.com/Unvanquished/Unvanquished/pull/3213